File: atcoder/abc054A_one_card_poker.ml
(* unihernandez22
 * https://atcoder.jp/contests/abc054/tasks/abc054_a
 * implementation
 * *)

Printf.printf "%s\n" @@
Scanf.scanf "%d %d\n" @@ fun a b ->
    if a == b then "Draw"
    else if a == 1 then "Alice"
    else if b == 1 then "Bob"
    else if a > b then "Alice"
    else "Bob"

File: atcoder/abc054A_one_card_poker.nim
# vicfred
# https://atcoder.jp/contests/abc054/tasks/abc054_a
# implementation
import strutils, sequtils

let line = stdin.readLine.split.map(parseInt)
let (a, b) = (line[0], line[1])

if a == 1 or b == 1:
  if a == b:
    echo "Draw"
  elif a == 1:
    echo "Alice"
  else:
    echo "Bob"
elif a > b:
  echo "Alice"
elif a < b:
  echo "Bob"
else:
  echo "Draw"

